### PR TITLE
add-back arrow backend tests

### PR DESCRIPTION
**Pull request by zhouyuan**
## What changes were proposed in this pull request?
This patch adds selected TPCDS queries for arrow backend.
Signed-off-by: Yuan Zhou <yuan.zhou@intel.com>

## How was this patch tested?

pass GHA

### CODE REVIEW

File: .github/workflows/unittests.yml [hunk 1]
```diff
@@ -22,6 +22,36 @@ on:
...
+      - run: sudo apt-get update
+      - run: sudo apt-get install -y cmake ccache build-essential
+      - run: sudo apt-get install -y maven
+      - run: sudo apt-get install -y libboost-all-dev libcurl4-openssl-dev
+      - run: sudo apt-get install -y libssl-dev
+      - run: sudo apt-get install -y libz-dev
+      - run: sudo apt-get install -y llvm clang
+      - name: Install Gluten Arrow-backend
+        run: |
+          mvn clean package -Dhttps.proxyHost=child-prc.intel.com -Dhttps.proxyPort=913 -Pbackends-velox -DskipTests -Dbuild_cpp=ON -Dbuild_gazelle_cpp=ON -Dbuild_arrow=ON
```
Comment (jinchengchenghh) [inline .github/workflows/unittests.yml:46]: please set the build_gazelle_cpp=OFF in velox test

Comment (zhouyuan) [reply .github/workflows/unittests.yml:46]: This is for arrow backend, so it should be ON

Comment (jinchengchenghh) [reply .github/workflows/unittests.yml:46]: **velox-backend-test:**
    runs-on: sr404-self-hosted
    steps:
I mean in velox, no need to double build arrow

Comment (zhouyuan) [reply .github/workflows/unittests.yml:46]: fixed in 
9d19a69

**Review by marin-ma — APPROVED** [review]

**Review by jinchengchenghh — APPROVED** [review]

**Pull request merged**